charts/mattermost-enterprise-edition/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Enterprise server with high availitibity.
 name: mattermost-enterprise-edition
-version: 1.5.0
+version: 1.6.0
```

```suggestion
version: 1.5.1
```

this is more a bug fix than a new feature